Modules/unicodedata.c

```diff
@@ -1388,10 +1543,17 @@ unicodedata_UCD_lookup_impl(PyObject *self, const char *name,
         return NULL;
     }
 
-    if (!_getcode(self, name, (int)name_length, &code, 1)) {
+    if (!_getcode(name, (int)name_length, &code)) {
         PyErr_Format(PyExc_KeyError, "undefined character name '%s'", name);
         return NULL;
     }
+    if (UCD_Check(self)) {
```

Comment:
Just making sure I understand: There is no behavior change because the old code has call graph `_getcode->_cmpname->_getucname` and `_getucname` has a related `if (UCD_Check(self)) {` block. Is that right?